src/utils/scalar-multiply.ts

```diff
@@ -40,6 +46,10 @@ export const scalarMultiplyWasmFallbackToJavascript = (
     if (!(err instanceof Error)) {
       throw err;
     }
+    if (err.message.includes('invalid y coordinate')) {
+      // Noble/ed25519 would also throw this error, so no need to call Noble
+      throw err;
```

Comment:
Note: I tested on device (with a wallet with legacy commitments) that this works.

Reply:
We might want to add a test case to engine tests that hits this statement, in case the string error from noble changes?

Reply:
Oh, here we are checking for the error message thrown from scalarmult-rsjs, not from Noble

Reply:
Hmm, one problem though is that while scalarmult-rsjs throws this error, scalarmult-wasm doesn't. I'll update scalarmult-wasm now too.

Reply:
Alright, updated

Reply:
The WASM (web) package now throws JS Errors instead of panicking (crashing) for two cases:
- [invalid y coordinate](https://github.com/Railgun-Community/curve25519-scalarmult-wasm/commit/524b3122e7f70ec5a4c2d199a3d458819fc4afae)
- [invalid scalar size](https://github.com/Railgun-Community/curve25519-scalarmult-wasm/commit/7b00b205c89c1029e4236b66b240068c77e9286a)

The Rust (mobile) package also throws "invalid y coordinate".